### DISCUSSION TITLE
Sending e-mail to attendees doesn't work, automatically or manually

### DISCUSSION THREAD

**Discussion by vb-03** [pretix/pretix#4642] [Q&A]
### Problem and impact

When confirming an order or sending out e-mails to individual attendees, only the order contact receives an
e-mail.

In this option, the fallback doesn't work - it just doesn't send the e-mails.

### Expected behaviour

Every attendee receives an e-mail.
### Steps to reproduce

1. Enable the "Send an email to attendees" in the event e-mail settings
2. Place an order 
or, 
2. When trying to send out e-mails, select the option "Every attendee". (In this option, the fallback doesn't even work - it just doesn't send e-mails.)

### Screenshots

![image](https://github.com/user-attachments/assets/c7e84f89-c8e1-42e3-9820-103389cf8e6c)

### Browser (software, desktop or mobile?) and version

Edge 129, Windows Desktop

### Operating system, dependency versions
Client: Windows 11
Server: Ubuntu 22.04.2 LTS, using NGINX as a reverse proxy and Docker

### Version

2024.9.0

**Comment by raphaelm** [comment]
Is your pretix instance not sending *any* emails, or just not the ones with the "send e-mails to all" feature?

**Comment by vb-03** [comment]
> Is your pretix instance not sending _any_ emails, or just not the ones with the "send e-mails to all" feature?

It sends e-mails to the order e-mail - automatically and through the "send e-mails to all" feature.

E-mails to attendees - automatically or through the "send e-mails to all" feature - just doesn't work.

Couldn't find anything in Docker logs as well.

E-mail provider for the organizer is Google SMTP Relay service.

**Comment by raphaelm** [comment]
Could you show a screenshot of your product list and of the detail view of an order? Feel free to blur out any personal data or order numbers, but the labels would be interesting.

**Comment by vb-03** [comment]
![image](https://github.com/user-attachments/assets/f3bd2e16-9f39-40df-ad71-44fab1e840cd)



![image](https://github.com/user-attachments/assets/0c0197fd-34bb-4916-8b04-178b2e3b3043)


I also blurred out some products which are not used in this case.

**Comment by raphaelm** [comment]
Ah, okay, this is a not-well-documented limitation: Emails are never sent to holders of add-on products, and your main-level products do not collect attendee addresses.

**Comment by vb-03** [comment]
Oh, okay, that makes sense.
Unfortunately, there isn't any other way I could do this - we're basically choosing a room here, and depending on the chosen room, it will have to add 4 or 6 people - and each person can choose to pay the full price or the member price if applicable.

Is this limitation ever going to be fixed?
Is it enough to collect e-mail on the main products?
